src/config/dynamic-theme-fixes.config

```
@@ -4379,6 +4379,19 @@ INVERT
 
 ================================
 
+distrowatch.com
+
+INVERT
+img[src="images/cpxtu/dwbanner.png"]
+
+CSS
+.NavMenu, 
+td:not(.NewsDate, .NewsHeadline) {
```

As per MDN:

```
Using two selectors at the same time is not yet supported in all browsers. Example: :not(.foo, .bar). For wider support you could use, :not(.foo):not(.bar)
```
So let's not hop into fancy features, and revert to a safe default:

```suggestion
td:not(.NewsDate):not(.NewsHeadline) {
```

---

Ah I didn’t know that, thanks for the heads up.